Gemfile

```
@@ -13,5 +13,5 @@ gem "minitest", "~> 5.11"
 gem "mocha", "~> 1.11", require: false
 gem "rack-test", "~> 2.0"
 gem "rake"
-gem "rubocop", "~> 0.80"
+gem "rubocop", "~> 1.35"
```

Since the latest rubocop requires ruby 2.6+ (it still targets/analyzes older rubies) and since CI only runs rubocop with ruby "3", I _think_ the following should fix the build:

```suggestion
gem "rubocop", "~> 1.35", platforms: %i[ruby_30 ruby_31]
```